### ISSUE TITLE
how to cite

### ISSUE THREAD

**Issue by jerabaul29** [OpenDrift/trajan#97]
Should there be a whitepaper / technical paper (JOSS?) to cite to give credit to TrajAn?

**Comment by gauteh** [comment]
Yes. We have been thinking about this, but maybe felt that there wasn't enough meat on the bones yet. A JOSS paper is probably a good idea if we want something fast, it takes less time than regular ones. We have most of the requirements ready: examples, documentation, versioning, packaging.

**Comment by gauteh** [comment]
You should absolutely be a co-author!
